spectrum_utils/plot.py

```diff
@@ -227,6 +232,111 @@ def spectrum(
     return ax
 
 
+def mass_errors(
```

Comment:
**suggestion:** Can you add an option to plot the mass errors with either ppm or m/z units?

Reply:
Indeed, I forgot about that! I was wondering when mass errors are Da vs ppm. Does this depend on the arguments passed to the annotation function? Can I assume that within a spectrum all mass errors use the same unit?

Reply:
The mass difference of the annotated fragments indeed depends on the fragment tolerance mode that was provided to `annotate_proforma`, and all mass differences for a single spectrum are the same.

So we could either:
1. Assume that the user specifies the fragment tolerance mode that they want during peak annotation, and we just display the corresponding mode on the axis label.
2. Provide a method to specify the fragment tolerance mode here as well. This would potentially require converting between m/z and ppm (or vice versa), so is a little bit more work.

I don't immediately have a strong preference. Option 1 gives us cleaner code, whereas option 2 is a bit more flexible.

Reply:
I added checks to find the mass error for a given `spec.annotation`, then convert the _m/z_ delta's if needed.